### ISSUE TITLE
Cluster create problem on new version script

### ISSUE THREAD

**Issue by Sivolen** [bb-Ricardo/netbox-sync#89]
Hi, i tested a new version and i have a problem:
Script mark all vm, servers, cluster how Orphaned and i have error.
```



2021-09-03 10:05:23,326 - INFO: Creating new NetBox 'cluster' object 'ykr-sub-srv01' with data: {'tags': [{'name': 'NetBox-synced'}, {'name': 'Source: DC-SRG3'}], 'name': 'ykr-sub-srv01'}
2021-09-03 10:05:23,399 - ERROR: NetBox returned: POST /api/virtualization/clusters/ Bad Request
2021-09-03 10:05:23,399 - ERROR: NetBox returned body: {'type': ['This field is required.']}
2021-09-03 10:05:23,399 - ERROR: Request Failed for cluster. Used data: {'tags': [{'name': 'NetBox-synced'}, {'name': 'Source: DC-SRG3'}], 'name': 'ykr-sub-srv01'}
2021-09-03 10:05:23,400 - INFO: Creating new NetBox 'cluster' object 'mak-upr-srv01' with data: {'tags': [{'name': 'NetBox-synced'}, {'name': 'Source: DC-SRG3'}], 'name': 'mak-upr-srv01'}
2021-09-03 10:05:23,472 - ERROR: NetBox returned: POST /api/virtualization/clusters/ Bad Request
2021-09-03 10:05:23,472 - ERROR: NetBox returned body: {'type': ['This field is required.']}
2021-09-03 10:05:23,472 - ERROR: Request Failed for cluster. Used data: {'tags': [{'name': 'NetBox-synced'}, {'name': 'Source: DC-SRG3'}], 'name': 'mak-upr-srv01'}
2021-09-03 10:05:23,473 - INFO: Creating new NetBox 'cluster' object 'ys-odb-srv01' with data: {'tags': [{'name': 'NetBox-synced'}, {'name': 'Source: DC-SRG3'}], 'name': 'ys-odb-srv01'}
2021-09-03 10:05:23,550 - ERROR: NetBox returned: POST /api/virtualization/clusters/ Bad Request
2021-09-03 10:05:23,550 - ERROR: NetBox returned body: {'type': ['This field is required.']}
2021-09-03 10:05:23,551 - ERROR: Request Failed for cluster. Used data: {'tags': [{'name': 'NetBox-synced'}, {'name': 'Source: DC-SRG3'}], 'name': 'ys-odb-srv01'}
2021-09-03 10:05:23,551 - INFO: Creating new NetBox 'cluster' object 'ugl-upr-srv01' with data: {'tags': [{'name': 'NetBox-synced'}, {'name': 'Source: DC-SRG3'}], 'name': 'ugl-upr-srv01'}
2021-09-03 10:05:23,624 - ERROR: NetBox returned: POST /api/virtualization/clusters/ Bad Request
2021-09-03 10:05:23,624 - ERROR: NetBox returned body: {'type': ['This field is required.']}

```

```
2021-09-03 10:05:24,744 - ERROR: Unfortunately updated item cluster for object anv-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'anv-upr-srv01' at 139790070854176>
2021-09-03 10:05:24,745 - ERROR: Unfortunately updated item cluster for object as-pol-srv01.mzso.local could not be fully resolved: <NBCluster instance 'as-pol-srv01' at 139790071225264>
2021-09-03 10:05:24,745 - ERROR: Unfortunately updated item cluster for object dol-dp-srv01.mzso.local could not be fully resolved: <NBCluster instance 'dol-dp-srv01' at 139790086074272>
2021-09-03 10:05:24,745 - ERROR: Unfortunately updated item cluster for object hlm-upr-srv02.mzso.local could not be fully resolved: <NBCluster instance 'hlm-upr-srv02' at 139790065487728>
2021-09-03 10:05:24,745 - ERROR: Unfortunately updated item cluster for object hlm-upr-srv03.mzso.local could not be fully resolved: <NBCluster instance 'hlm-upr-srv03' at 139790083100432>
2021-09-03 10:05:24,745 - ERROR: Unfortunately updated item cluster for object kur-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'kur-upr-srv01' at 139790062874976>
2021-09-03 10:05:24,745 - ERROR: Unfortunately updated item cluster for object mak-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'mak-upr-srv01' at 139790055534704>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object ngl-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ngl-upr-srv01' at 139790062690848>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object oha-pol-srv01.mzso.local could not be fully resolved: <NBCluster instance 'oha-pol-srv01' at 139790078673344>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object prn-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'prn-upr-srv01' at 139790070839520>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object skr-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'skr-upr-srv01' at 139790067311424>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object tmr-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'tmr-upr-srv01' at 139790066978336>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object tym-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'tym-upr-srv01' at 139790086453520>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object ugl-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ugl-upr-srv01' at 139790067312576>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object ykr-sub-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ykr-sub-srv01' at 139790054530496>
2021-09-03 10:05:24,746 - ERROR: Unfortunately updated item cluster for object ykr-upr-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ykr-upr-srv01' at 139790083100624>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-dgp-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-dgp-srv01' at 139790066976608>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-doms-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-doms-srv01' at 139790084994048>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-gp2-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-gp2-srv01' at 139790086072496>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-gp4-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-gp4-srv01' at 139790070962544>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-kdc-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-kdc-srv01' at 139790086150176>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-kvd-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-kvd-srv01' at 139790053967424>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-odb-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-odb-srv01' at 139790091118768>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-okb-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-okb-srv01' at 139790067012848>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-okb-srv02.mzso.local could not be fully resolved: <NBCluster instance 'ys-okb-srv02' at 139790083165584>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-opb-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-opb-srv01' at 139790071126144>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-opb-srv02.mzso.local could not be fully resolved: <NBCluster instance 'ys-opb-srv02' at 139790054528480>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-osp-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-osp-srv01' at 139790067403744>
2021-09-03 10:05:24,747 - ERROR: Unfortunately updated item cluster for object ys-vfd-srv01.mzso.local could not be fully resolved: <NBCluster instance 'ys-vfd-srv01' at 139790083720480>

```

**Comment by Sivolen** [comment]
I can send all log or debug log

**Comment by bb-Ricardo** [comment]
Are you using the `srip_host_domain_name` option. And are these stand alone hosts?

**Comment by Sivolen** [comment]
Hi, i don't use this option, No it's several  servers

**Comment by bb-Ricardo** [comment]
Can you send me a log where this happens as well please? Don't really understand why this is happening. Cause your host should be considered a "standalone" host but the log (you sent for the other issue) doesn't mention it.

**Comment by Sivolen** [comment]
Hi, ok, I'm sending on Monday, now I've rolled back to the previous version, everything works well on it

**Comment by bb-Ricardo** [comment]
alright. Thank you

**Comment by bb-Ricardo** [comment]
Ok, found the issue 🙈 . Can you try the latest commit and see if it works? Thank you very much.

**Comment by Sivolen** [comment]
Hi, this version working great, thanks )

**Comment by bb-Ricardo** [comment]
fixed in version v1.1.2!
